README.md

```diff
@@ -84,12 +84,16 @@ ApplePaySessionPolyfill.createPaymentToken = function (session) {
+      signature: "AAAAAAaAAAa9AAAAAaAAAAAAAAAaAaAAAaaaaaaAAAAAAaAAAAAAAaaaaaaA9a9AAaAAAAAAAAAA9aAAA9aaAaAAAaAAAAAAAAAaAAAaAaAAAaAAaa9AAaAaaaAaAAaAA9AAAaaaAAAaaAAaAAAaaAaaAAAaa99aAA99AAaaAAAaa99aA9AaAAAAAaAaAAAAA9AAAaaaAAAaaAAaA9AaaAaaaAAaaAaaaaAAaAAaa9AaaAaaAaAAAaAAAAaAAaAaaAaaAAaaAa9aAaAAAaAAAAAAAaAAAA9AAAA9AAAaAAAaAaA9A9aAAAA9AAAaAaAaAaA9A9aaAaAaAAAAA9AAAaaaAAAaAAAaaA9aaa9aAAAaa9aaaa9AAaAaAAAAAAAAAAAAA9AAAaaAaA9AAAA9a9AaaAAaAaAAAaAAAAaAAaAaaAaaAAaaAa9aAaAAAaAAAAAAAaAAAAaaAaAAAaAAaa9AAAAAAaAAaa9AAAaAAaAAaaA99aaAa9Aaa9aaaAAaAA9AaA9aAAa9aAAA9Aa9aAAaa9AA9AAaAA9AAa9aAAA99aAa9aaA9A/9aaaAAaaAA9AAAaAaaaAAAAaAA9AaAaAA/aAAAAAaAaAAAA9aAAaaAaAAA/AAaA+A9A9a9aA9AAa/aaa9AaaaAAAAAaAAAAAAAAAAAAA9AAAAAAaAAAAAAaAAaaaaaAAaAa9aa9AaaA9aaAAaAA9aa99aa9AaaAA9AAAaaAaaAAaaAAAaAaAAAA9AA9AaAAAAAAAaaaAAAAAAAAAAAaAAaaaaAAAAAAA+AAAAAaaaAaAAAAaAAaAAaaaAa9AaaAaaaaAaAA9aAAAaaAAaA9AaaAaaaAAaaAAaAaaaAA99AAAaaaA9AAAaa9AaAAAaAAAaAAA9AA9aAAAaAaA9aAAaaAaaaaAaaAAaaAAaAaaaAAA9AA9aAAAaAAAaaa9aAAAaAAAaa99aaAAaa99aAA9aAAAaAAaaA9AaaAaaaAAaaAAaaA9aaAA9AAAaAAAaAAA9aAAaA9A9aA9aAAAaAAA9aAAaAAA9AAAaaAAaaAAaAAAAAAaAAAAAAaAAAaaaaAAaAa9aa9a9AaAaaAaaAaAaaA9aAAA9aAAaA9A9AAA9aAaaaaa9aA9aAAAAAA9aAA9aAaAaaAaaAAAaaAA9aAaaA9AaaA9aaAAaAA9aa99aAAAaaAAaaAAaAa9aaaaaAAAAAA9AAAAAAAAA99/AaAAAaAa9A99AaAaaa9aaAA9AA9AaAaAA/aAAAaAAaAAAAaaaaaaA99AaAa9AAaAAAAaAAAaAAA99AAAAA9aAAAAAAAA+AAaa9a9aa9aA9a+aaAaa99AaAAaAAaaaAAA9Aa9aAAAaAAAAAAaa9a9aaAAaAaAaaa9Aa9Aaa9a9a9AAAaaa9aaaAAAA9aAAAaAaAaAAAaAAAA9aaaaA9aaaAaAAAaAAaa9AAaAaAaAaAAaAA9AAAaaAAAAaaAAaAa9aaAAAAAAaAAaaAAAaAAAAAAAAAA9AaAAaAAAAAAA9aAAaA9A9aA9aAAA9aAaaaaa9aAAAAAAAA9AAAaaAAAAaaAAaAA9aAaAAAAaAA9AAAaAAAAAaAaaAAAAaAAA9AaA9AaAaAaaAAaaaAAA9AaA9AaAaAaA9AA9aAAAAAAAAAAAAaAAaAAAAaAAaaAAaaAaaaaAAaaAaA9AaaAaaaaAAAAAaAAaaAAAaAAAAAAAAAA9AaAAaAAAAAAA9aAAaA9A9aA9aAAA9aAaaaaa9aAAAAAAAA9AAAaaAAAAaaAAaAA9aAaAAAAaAA9AAAaAAAAAaAAAAAaaaaaaAAAAAAaaaaaaAAAAAAaAAAAAaAaAAAaaaaaAaAaAAA9aaaAAa9aaaAaAA/AaA9AAA9AAaAa9a9Aa9AAAAa9aA/AAAaaaAaaaaaA9A9AaA/9AAa9A9AAA9AAAAAAaAAAAAAaAAAAaaAAA9AaaaAaAAAAaaAAAaaAA9aAaaA99aa9AaAAAaaAAaA99aA99aa9AaAA9aaAAaAAAaa9AaAAaaAA9AA9AaAaAAAAAa9aaAA9Aa9aaaaAAaaA+aa/9aAaAAAaAAAAAAAa9AAAAAAAA/AA9AA9AaAaAAAAaAAAaa9aAAA9aaaAaA9a9999/aaaAaAAaAA9AaAaAaAA9aAAAaaAaAAaa9aAA9Aa9aaaaaAAAaaAAaA99aA9AaaAaaaa9aaAAaAaAaA9AaAA9AA9AaAaAA/aAAAaAAAaAAAaaaaaaA99AaAaAAAAAAAAAAAaaaaaaAAAAAAaAaAAAaAaA9a9AAAAaAaAa9AaAAaaA/9Aaa9A99AaAAaaAaaA9Aa9a9aAaaAA+A9AAAa99aaa9AAAAa9AAaaAAAa9aa9aa9aAa9A9aaAA9AA9AaaAAAa9A9aa9aaaaaAAaaAAaAAaaAAAAAAAAAaAAAAAaAAAAaaAAaaaAaAAaAA9AAAaaaAAAaaAAaAAAaaAaaAAAaa99aAA99AAaaAAAaa99aA9AaAAAAAaAaAAAAA9AAAaaaAAAaaAAaA9AaaAaaaAAaaAaaaaAAaAAaa9AaaAaaAaAAAaAAAAaAAaAaaAaaAAaaAa9aAaAAAaAAAAAAAaAAAaaAAAAAAA9AAaAAAaaaaaaAAAAAAaAAAAAAaAAAAaaaaaaA9a9AAAAaAaAAAaAAaaaAAAaAAAaAAAaAAAa9AAAAAAAAAa9aAAA9AaAaAAAaAAAaAAaAAAaAAAa9AAAAAAAaAAaaAAAAAAAAAAAAAAAAAAAaAaAAAaAAaa9AAaAaAaAAAaAAaaaAAAaAAAAAAA9Aa9aAAAAA9AAaAaA9a9aA9AAaaaA9AaAaaaaA/9AAAAaAAAaAAA99AAAAAAaaAAAaAA9aaAAAaAaaAA9AAAaaaAAaAaAA9Aa9AAaAAaaaAaAaAaAAAaAA+a9A9AAaaaaaaaaA9a9aaaAAa+AAA9Aaa99aAAAAAAAAAA==",
```

Comment:
Do these need to have quite so much data in them? It introduces quite a big horizontal scrollbar.

Reply:
Actually I'm not sure. haha
I just redacted keeping letters and numbers of the original.

But yes... I agree it's maybe too much.